commerce-payment-web/src/main/java/com/liferay/commerce/payment/web/internal/display/context/CommercePaymentMethodGroupRelsDisplayContext.java

```diff
@@ -223,6 +290,41 @@ public String getSelectedScreenNavigationCategoryKey() {
 				CATEGORY_KEY_COMMERCE_PAYMENT_METHOD_DETAILS);
 	}
 
+	public int getSubscriptionEntryCount(String engineKey)
+		throws PortalException {
+
+		ThemeDisplay themeDisplay = (ThemeDisplay)_renderRequest.getAttribute(
+			WebKeys.THEME_DISPLAY);
+
+		int activeSubscriptions = 0;
+
+		long groupId =
+			_commerceChannelLocalService.getCommerceChannelGroupIdBySiteGroupId(
+				themeDisplay.getSiteGroupId());
+
+		List<CommerceOrder> commerceOrders =
+			_commerceOrderLocalService.getCommerceOrders(groupId, engineKey);
+
+		for (CommerceOrder commerceOrder : commerceOrders) {
```

Comment:
for inside a for, I would always prefer to avoid this scenario, make code hard to follow